sound/soc/sof/ipc4-topology.c

```diff
@@ -1582,55 +1582,92 @@ bool sof_ipc4_copier_is_single_bitdepth(struct snd_sof_dev *sdev,
 	return true;
 }
 
+static int
+sof_ipc4_narrow_params_to_format(struct snd_sof_dev *sdev,
```

Comment:
"narrow params" does not read well, it can be a verb or an adjective. Did you mean 'adjust' or 'limit'?

Reply:
Right, "adjust" is better, thanks